docs/src/architecture/09_architecture_decisions/0005-rust.md

```diff
@@ -0,0 +1,84 @@
+---
+    title: 0005 Rust
+    adr:
+        author: Steven Johnson
+        created: 09-Jan-2024
+        status:  accepted
+    tags:
+        - Rust
+---
+
+## Context
+
+Languages used in the project need to be chosen based on:
+
+* their strengths
+* suitability to the tasks they are used for
+* familiarity with the team.
+
+## Assumptions
+
+* That the language chosen will continue to be a mainstream language.
+* That it will be reasonably easy to find developers proficient in it.
+
+## Decision
+
+Backend and System, level services will be coded in:
+
+* Rust
```

Comment:
If the backend extends to mean anything like an API, then I think this is overly strict. Even "service" is really vague. Would the archiver fall under this category?